installer/templates/phx_assets/webpack.config.js

```diff
@@ -45,9 +36,19 @@ module.exports = (env, options) => {
       ]
     },
     plugins: [
-      new MiniCssExtractPlugin({ filename: '../css/[name].css' }),
-      new CopyWebpackPlugin([{ from: 'static/', to: '../' }])
-    ]
-    .concat(devMode ? [new HardSourceWebpackPlugin()] : [])
+      new MiniCssExtractPlugin({ filename: '../css/app.css' }),
+      new CopyWebpackPlugin({
+        patterns: [
+          { from: 'static/', to: '../' }
+        ]
+      })
+    ],
+    optimization: {
+      minimizer: [
+        '...',
```

Comment:
```suggestion
        `...`,
```
This should use backticks.

Reply:
Why should this use JS template literal?

Reply:
The CssMinimizerWebpackPlugin [documentation](https://webpack.js.org/plugins/css-minimizer-webpack-plugin/) says so, but I haven't actually confirmed if single quotes work too.

Reply:
I think they just meant to use markdown syntax there, it has no further meaning as far as I know, the docs on this page just use regular quotes:

https://webpack.js.org/blog/2020-10-10-webpack-5-release/

![image](https://user-images.githubusercontent.com/1214337/100892089-3dc7d580-34ba-11eb-86a1-6d6c39c5e0b3.png)

Reply:
Ahh ok. That's fine. I saw the documentation too. They also miss the `,` colon there (without it's not working).
So of course it'll work with single quotes too.

There is no need for backticks (JS template literal) - they are used for e.g. `Hello ${name}`.

Reply:
Thanks for clarifying that :relaxed: